main.rb

```diff
@@ -0,0 +1,76 @@
+require './app'
+
+class Main
+    def initialize
+      @app = App.new
+      @output = 0
+    end
+    def options
+      print  "WELCOME TO THE SCHOOL LIBRARY APP"
+      while @output != 7
+        print "\nPlease choose an option by entering a number \n\n"
+        print "1 - List all books \n"
+        print "2 - List all people \n"
+        print "3 - Create a person (teacher or student) \n"
+        print "4 - Create a book \n"
+        print "5 - Create a rental \n"
+        print "6 - List all rentals for a person \n"
+        print "7 - Quit \n"
+        start_app
+       end 
+      end
+
+      def start_app
+        @output = gets.chomp.to_i
+        case @output
+        when 1
+          @app.list_books
+        when 2
+          @app.list_people
+        when 3
+          @app.create_person
+        when 4
+          @app.create_book
+        when 5
+          @app.create_rental
+        when 6
+          @app.list_rentals
+        end
+      end     
+end
+
+main = Main.new
+main.options
+
+
+
+# def functional(app)
+#   user_input = options
+#   case user_input
+#   when 1
+#     app.
+#     functional(app)
+#   when 2
+#     app.list_people
+#     functional(app)
+#   when 3
+#     app.create_person
+#     functional(app)
+#   when 4
+#     app.create_book
+#     functional(app)
+#   when 5
+#     app.create_rental
+#     functional(app)
+#   when 6
+#     app.list_rentals
+#     functional(app)
+#   end
+# end
+
+# def main
+#   app = App.new
+#   functional(app)
+# end
+
+# main
```

Comment:
### **Optional**
- [ ] Please remove all commented code from your codebase to make your code clean and easy to maintain.

Reply:
done as requested